llama_hub/tools/cogniswitch/README.md

```diff
@@ -0,0 +1,71 @@
+## Cogniswitch ToolSpec
+
+**Use cogniswitch to have chat with your knowledge in just three steps**
+
+visit https://www.cogniswitch.ai/developer.<br>
+
+**Registration:**
+- Signup with your email and verify your registration
+- You will get a mail with a platform token and oauth token for using the services.
+
+
+**step 1: Instantiate the Cogniswitch ToolSpec:**<br>
+- Use your cogniswitch token, openAI API key, oauth token to instantiate the toolspec. <br> 
+
+**step 2: Cogniswitch Store data:**<br>
+- use store_data function in the toolspec and input your file or url. <br>
+- it will be processed and stored in your knowledge store. <br> 
+- you can check the status of document processing in cogniswitch console. <br>
+
+**step 3: Cogniswitch Answer:**<br>
+- Use query_knowledge function in the toolspec and input your query. <br>
+- You will get the answer from your knowledge as the response. <br>
+
+
+### Import Required Libraries
+
+
+```python
+import warnings
+warnings.filterwarnings("ignore")
+from llama_hub import CogniswitchToolspec
```

Comment:
It should be like
`from llama_hub.tools import CogniswitchToolspec`

Reply:
thanks anoop will make the changes accordingly.